cypress/e2e/webTables.cy.js

```
@@ -1,7 +1,182 @@
+    cy.typeInputByPlaceholder(age, 'Age');
+    cy.typeInputByPlaceholder(salary, 'Salary');
+    cy.typeInputByPlaceholder(department, 'Department');
+    cy.clickSubmitButton();
+    cy.assertNewUser(worker);
+  });
+
+  it('should not accept letters in salary input in submit form', () => {
+    cy.get('#edit-record-1')
+      .click();
+    cy.typeInputByPlaceholder('2000aaddd', 'Salary');
+    cy.clickSubmitButton();
+    cy.get('.modal-body')
+      .should('be.visible');
+  });
+
+  it(`should not accept email without dot, '@' and spaces`, () => {
+    cy.get('#edit-record-1')
+      .click();
+    cy.typeInputByPlaceholder('Cierra@examplecom', 'name@example.com');
+    cy.clickSubmitButton();
+    cy.get('.modal-body')
+      .should('be.visible');
+    cy.typeInputByPlaceholder('Cierra@example     com', 'name@example.com');
+    cy.clickSubmitButton();
+    cy.get('.modal-body')
+      .should('be.visible');
+    cy.typeInputByPlaceholder('Cierraexample.com', 'name@example.com');
+    cy.clickSubmitButton();
+    cy.get('.modal-body')
+      .should('be.visible');
+  });
+
+  it('should find workers for all column values', () => {
+    cy.countWorkers(allWorkers)
+      .then(() => {
+        const lastWorker = allWorkers[allWorkers.length - 1];
+
+        cy.assertResultOfSearchedWorker(lastWorker.firstName);
+        cy.assertResultOfSearchedWorker(lastWorker.lastName);
+        cy.assertResultOfSearchedWorker(lastWorker.age);
+        cy.assertResultOfSearchedWorker(lastWorker.email);
+        cy.assertResultOfSearchedWorker(lastWorker.salary);
+        cy.assertResultOfSearchedWorker(lastWorker.department);
+      });
+  });
+
+  it('should adjust rows count selection', () => {
+    const rowsSelection = [
+      '5', '10', '25', '50', '100'
+    ];
+
+    rowsSelection.forEach((option) => {
+      cy.selectAmountOfRows(option);
+      cy.assertAmountOfRows(option);
+    });
+  });
+
+  it.only('should have worked pagination', () => {
```

Using `.only` is a useful tool for debugging, but it must be removed from the final code. Leaving it in prevents the rest of the tests in the suite from running, which can hide failures in other parts of the test file.